package.json

```diff
@@ -11,10 +11,12 @@
     "bootstrap": "^4.3.1",
     "dotenv": "^8.0.0",
```

Comment:
Note to ask about the appropriate handling of the Mapbox token.

Reply:
I think we might want to add URL restrictions: https://docs.mapbox.com/help/how-mapbox-works/access-tokens/

Reply:
opened an issue for this... #35